.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: java
 
 sudo: false
+dist: precise
+
+before_install:
+  - export M2_HOME=/usr/local/maven
```

Comment:
Manually installing maven, no. But maven command exists, we should be able to do somrthing like `export M2_HOME=$(which mvn)`.